src/Data/Str.php

```diff
@@ -21,23 +21,39 @@
 class Str implements StrContract
 {
     /**
-     * Reverse the string.
+     * Get the encoding.
      *
-     * @param string $str String to be evaluated.
+     * @param string $encoding Valid encoding.
      *
      * @since 3.0.0
      *
      * @return string
      */
-    public static function reverse(string $str) :string
+    private static function encoding($encoding = null) :string
     {
-        if (function_exists('strrev')) {
-            return strrev($str);
-        }
+        return $encoding ?: \mb_internal_encoding();
```

Comment:
The `mb_internal_encoding` function is available on `php-mbstring` extension.

It should check the `mbstring` is available during composer installation.

Reply:
If not available we have to install it?

I think you may send PR, in your available time